packages/components/src/components/typography/typography.stories.tsx

```diff
@@ -10,75 +10,75 @@ export default meta;
 export const All: StoryFn = () => (
     <>
         <StoryColumn>
-            <H1 data-test="heading-1">Heading 1</H1>
-            <H2 data-test="heading-2">Heading 2</H2>
+            <H1 data-test-id="heading-1">Heading 1</H1>
```

Comment:
@jvaclavik why we have `data-test-id` in storybook?

Reply:
I noticed it too, and was perplexed because those ids in `*.stories.tsx`  are not referenced anywhere. Seems useless, but I refactored it anyway for consistency.

Reply:
It doesn't make sense to me, let's drop it.

Reply:
I think that long time ago there were cypress tests configured to run also for the storybook.